doc/swagger.yaml

```diff
@@ -2927,6 +2947,53 @@ paths:
       summary: Updates a DID document.
       tags:
       - DecentralizedIdentityAPI
+  /v1/dids/{method}/{id}/deactivation:
```

Comment:
this can be more restful as a delete to `/v1/dids/{method}/{id}`

Reply:
hm, I see. awkward because we both need to handle the service's concept of deletion and the did methods
I'm inclined to merge the two...
Something lke -- if the DID method supports deactivation, the `DELETE` to `/v1/dids/{method}/{id}` does both delete and deactivation. If the DID method does not support deactivation, we just do a delete.

Reason being, it's more confusing to support both separately. What do you think?

Reply:
I like that, will update.